app/models/patient.rb

```diff
@@ -440,6 +441,49 @@ def latest_transfer
     end
   }
 
+  # Criteria for this CDC quarantine guidance which can be found here:
+  # https://www.cdc.gov/coronavirus/2019-ncov/more/scientific-brief-options-to-reduce-quarantine.html
+  #
+  # Record must:
+  # - be unpurged, open, in exposure workflow, and not in continuous exposure
+  # - have reported within 10-13 days after their last date of exposure and has no symptomatic reports
+  # - be between 10-13 days past their last date of exposure
+  scope :ten_day_quarantine_candidates, lambda { |user_curr_datetime|
+    where(purged: false, monitoring: true, isolation: false, continuous_exposure: false)
+      .where_assoc_exists(:assessments) do
+        # CAST is necessary to guarantee correct comparison between datetime and date.
+        where('CAST(assessments.created_at AS DATE) BETWEEN DATE_ADD(last_date_of_exposure, INTERVAL 10 DAY) '\
```

Comment:
I want to point out that this `CAST` will cause this query to ignore indexes related to this column I believe. I may need to update for performance reasons. I also would like to verify that this code won't break if we upgrade our MySQL version.

Reply:
What makes you think it ignores indexes?

Reply:
Haha you found it. Yeah the docs + also we ran into this when Tim did a `BINARY` on the submission tokens, so we had to change the column type so it would actually use indexes. This may be enough of a problem that I have to change this, will do performance testing soon. If you have the time to as well that'd be extremely appreciated